### ISSUE TITLE
Cannot find NAMESPACE file

### ISSUE THREAD

**Issue by jstevens5** [ropenscilabs/autotest#42]
I'm trying out the `autotest_package()` function on a package that I created, but it generates an error as soon as I run it. Specifically, it cannot find the `NAMESPACE` file. Even though the working directory in RStudio is set to the package root directory, it seems to be looking for `NAMESPACE` in the system R libraries directory.

```
x <- autotest_package (package = "foo")
Error in fns_from_other_pkgs(package) : 
  There is no NAMESPACE file at [/usr/lib/R/library/excluder]
```

I'm using this in Ubuntu 20.04, R 4.0.4, and RStudio 1.4.1106.

**Comment by jstevens5** [comment]
Ah, now I see that you need to omit the `package` argument  or use `package = "."` to use package in the current directory.

**Comment by mpadge** [comment]
Ah, I love an auto-resolving issue :grinning: Thanks @jstevens5, and feel free to open future issues - the package really needs testing out in the wild!

**Comment by jstevens5** [comment]
I'm writing my first package, and testing is definitely my weakness, so I'm looking forward to implementing autotest. Two things on this:
1. If other folks have this problem, hopefully they'll find this issue. But it might be helpful to have explicit examples of running `autotest_package()` on local packages in the documentation (e.g., in the `autotest_package()` documentation and maybe the README). 
2. I did receive another error as soon as I got past this. I'll look into this more in the coming week to see if I need to raise another issue.
